build/spas.ts

```diff
@@ -415,90 +420,37 @@ async function fetchRecentContributions() {
 }
 
 async function fetchLatestNews() {
-  const items: NewsItem[] = [];
-
-  items.push(
-    {
-      title: "Responsibly empowering developers with AI on MDN",
-      url: `https://blog.mozilla.org/en/products/mdn/responsibly-empowering-developers-with-ai-on-mdn/`,
-      author: "Steve Teixeira",
-      published_at: new Date("Thu, 06 Jul 2023 14:41:20 +0000").toString(),
-      source: {
-        name: "blog.mozilla.org",
-        url: `https://blog.mozilla.org/en/latest/`,
-      },
-    },
-    {
-      title: "Introducing AI Help: Your Trusted Companion for Web Development",
-      url: `/${DEFAULT_LOCALE}/blog/introducing-ai-help/`,
-      author: "Hermina Condei",
-      published_at: new Date("2023-06-27").toString(),
-      source: {
-        name: "developer.mozilla.org",
-        url: `/${DEFAULT_LOCALE}/blog/`,
-      },
-    },
-    {
-      title: "Introducing the MDN Playground: Bring your code to life!",
-      url: `/${DEFAULT_LOCALE}/blog/introducing-the-mdn-playground/`,
-      author: "Florian Dieminger",
-      published_at: new Date("2023-06-22").toString(),
-      source: {
-        name: "developer.mozilla.org",
-        url: `/${DEFAULT_LOCALE}/blog/`,
-      },
-    },
-    {
-      title: "Introducing Baseline: a unified view of stable web features",
-      url: `/${DEFAULT_LOCALE}/blog/baseline-unified-view-stable-web-features/`,
-      author: "Hermina Condei",
-      published_at: new Date("2023-05-10").toString(),
-      source: {
-        name: "developer.mozilla.org",
-        url: `/${DEFAULT_LOCALE}/blog/`,
-      },
-    },
-    ...(await fetchHacksNews())
-  );
+  const items: NewsItem[] = (
+    await Promise.all(
+      LATEST_NEWS.map(async (itemOrUrl) => {
+        if (typeof itemOrUrl !== "string") {
+          return itemOrUrl;
+        }
+        const url = itemOrUrl;
+        const post = await findPostBySlug(
+          getSlugByBlogPostUrl(`/${DEFAULT_LOCALE}/${url}`)
+        );
+        if (post) {
+          const {
+            doc: { title },
+            blogMeta: { author, date, slug },
+          } = post;
+          return {
+            title,
+            url: `/${DEFAULT_LOCALE}/blog/${slug}/`,
+            author: author?.name || "The MDN Team",
```

Comment:
In future, we might want to extend this to something else or consider a different key, because we're missing posts from Hermina, Dipika, and Vadim, etc. but we can discuss that later, this is an obvious improvement for now.